ProjectShop/OrderItems/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from order.models import Order
+
+
+# Create your models here.
+class OrderItems(models.Model):
+    order_id = models.ForeignKey(Order, on_delete=models.CASCADE)
+    # product_id = models.ForeignKey(Product, on_delete=models.CASCADE)
+    quantity = models.PositiveIntegerField(verbose_name="quantity")
```

Comment:
It would be good to add a translation:
from django.utils.translation import gettext as _
quantity = models.PositiveIntegerField(verbose_name=_("quantity"))
Please, check this link for more information:
https://docs.djangoproject.com/en/3.2/topics/i18n/translation/